Makefile.ide

```diff
@@ -123,6 +123,15 @@ ide/%.cmx: ide/%.ml
 	$(SHOW)'OCAMLOPT  $<'
 	$(HIDE)$(OCAMLOPT) $(COQIDEFLAGS) $(OPTFLAGS) -c $<
 
+# We need to compile this file without -safe-string due mess with
+# lablgtk API. Other option is to require lablgtk >= 2.8.16
```

Comment:
Does Travis test the compilation with both lablgtk versions (i.e. the minimum we require and the latest) ?

Reply:
I think it does not, will add some bounds.

Reply:
The PR seems ready to go, [it build correctly on my Travis], the bounds are in place.